cea/datamanagement/zone_helper.py

```
@@ -15,10 +15,9 @@
 
 import cea.config
 import cea.inputlocator
-from cea.datamanagement.databases_verification import COLUMNS_ZONE_TYPOLOGY
+from cea.datamanagement.databases_verification import COLUMNS_ZONE_GEOMETRY, COLUMNS_ZONE
```

Ruff (0.8.2)

`cea.datamanagement.databases_verification.COLUMNS_ZONE_GEOMETRY` imported but unused

Remove unused import: `cea.datamanagement.databases_verification.COLUMNS_ZONE_GEOMETRY`

(F401)